jekyll/_docs/installing-elasticsearch.md

```diff
@@ -4,41 +4,41 @@ title: Install a custom version of Elasticsearch
 description: Installing a custom version of Elasticsearch
 ---
 
-To enable Elasticsearch, add the following to your circle.yml:
+To enable Elasticsearch, add the following to your `circle.yml`:
 
 ```
 machine:
   services:
     - elasticsearch
 ```
 
-The default version of elasticsearch is {{ site.data.precise.versions.elasticsearch }}.
-If you need a custom version, you can download and start it from your build. To install 1.0.1, add the following to your circle.yml:
+The default version of Elasticsearch is {{ site.data.precise.versions.elasticsearch }}.
+If you need a custom version, you can download and start it from your build. To install 2.4.3, add the following to your `circle.yml`:
 
 ```
 dependencies:
   post:
-    - wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.0.1.tar.gz
-    - tar -xvf elasticsearch-1.0.1.tar.gz
-    - elasticsearch-1.0.1/bin/elasticsearch: {background: true}
+    - wget https://download.elastic.co/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.4.3/elasticsearch-2.4.3.tar.gz
+    - tar -xvf elasticsearch-2.4.3.tar.gz
+    - elasticsearch-2.4.3/bin/elasticsearch: {background: true}
     # Make sure that Elasticsearch is up before running tests:
     - sleep 10 && wget --waitretry=5 --retry-connrefused -v http://127.0.0.1:9200/
 ```
 
 <span class='label label-info'>Note:</span>
-remember to remove elasticsearch from machine.services if you install it manually.
+remember to remove elasticsearch from `machine.services` if you install it manually.
```

Comment:
Thanks! This was the first time I'd seen this way of writing it, and I didn't know if it was some standard secret dot notation I didn't know about. Will change.